File: jdt-dpgen-cli/src/main/resources/dpgen-output/src/main/scala/com/gof/behavioral/visitor/ConcreteElementB.scala
package com.gof.behavioral.visitor

//remove if not needed
import scala.collection.JavaConversions._

class ConcreteElementB extends Element {

  var counter: Int = _

  def operationElementB() {
    counter += 1
  }

  protected def getcounter(): Int = counter

  override def accept(visitor: Visitor) {
    visitor.visitConcreteElementB(this)
  }
}

File: jdt-dpgen-cli/src/main/resources/dpgen-output/src/main/java/com/gof/behavioral/visitor/ConcreteElementB.java
package com.gof.behavioral.visitor;
public class ConcreteElementB implements Element {

	public int counter;

	public void operationElementB() {
		counter++;
	}

	protected int getcounter() {
		return counter;
	}

	@Override
	public void accept(Visitor visitor) {
		visitor.visitConcreteElementB(this);
	}
}